### DISCUSSION TITLE
Installation issues on MacOS Big Sur on Macbook Pro M1

### DISCUSSION THREAD

**Discussion by mar-maradnus** [firefly-iii/firefly-iii#5187] [Firefly III] [answered]
Installing on MacBook Pro M1 running Big Sur. Have installed AMP and Composer. Installing Firefly with "composer create-project grumpydictator/firefly-iii --no-dev --prefer-dist firefly-iii 5.6.2" fails with

"Creating a "grumpydictator/firefly-iii" project at "./firefly-iii"
Installing grumpydictator/firefly-iii (5.6.2)
  - Installing grumpydictator/firefly-iii (5.6.2): Extracting archive
Created project in /Users/username/Sites/firefly-iii
> @php -r "file_exists('.env') || copy('.env.example', '.env');"
> @php -r "if (!(getenv('DYNO'))===false){file_exists('.env') || copy('.deploy/heroku/.env.heroku', '.env');copy('.deploy/heroku/.locales', '.locales');}"
Installing dependencies from lock file
Verifying lock file contents can be installed on current platform.
Your lock file does not contain a compatible set of packages. Please run composer update.

  Problem 1
    - nette/schema is locked to version v1.2.1 and an update of this package was not requested.
    - nette/schema v1.2.1 requires php >=7.1 <8.1 -> your php version (8.1.0RC5-dev) does not satisfy that requirement.
  Problem 2
    - nette/schema v1.2.1 requires php >=7.1 <8.1 -> your php version (8.1.0RC5-dev) does not satisfy that requirement.
    - league/config v1.1.1 requires nette/schema ^1.2 -> satisfiable by nette/schema[v1.2.1].
    - league/config is locked to version v1.1.1 and an update of this package was not requested.

Any pointers?

**Comment by JC5** [comment]
Don't use PHP 8.1 RC5-dev, use 8.0

**Comment by mar-maradnus** [comment, marked as answer]
That worked. Thanks.
